Doc/library/datetime.rst

```diff
@@ -2611,6 +2611,9 @@ differences between platforms in handling of unsupported format specifiers.
 .. versionadded:: 3.12
    ``%:z`` was added.
 
+.. versionchanged:: next
+   Digits must be ASCII
```

Comment:
```suggestion
.. versionchanged:: next
   Non-ASCII digits are now rejected by ``strptime``.
```

Or find something mentioning both strptime and the non ascii digits in the input. Because the current text doesn't say anything about whether it's for the directive or for something else and is confusing.